Taskfile.yml

```diff
@@ -45,15 +45,15 @@ tasks:
     deps:
       - task: poetry:install-deps
     cmds:
-      - poetry run codespell
+      - poetry run codespell --skip="node_modules"
 
   # Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/assets/spell-check-task/Taskfile.yml
   general:correct-spelling:
     desc: Correct commonly misspelled words where possible
     deps:
       - task: poetry:install-deps
     cmds:
-      - poetry run codespell --write-changes
+      - poetry run codespell --write-changes --skip="node_modules"
```

Comment:
Please use the `skip` field of the `.codespellrc` configuration file for this:
https://github.com/arduino/setup-task/blob/main/.codespellrc#L6

Reply:
Ops, I didn't see there was a `.codespellrc` file, I'm going to move them :+1: